### PR TITLE
save category buttons name in local storage

### PR DESCRIPTION
**Pull request by kakashii7**
Relate #7

### CODE REVIEW

**Review by A7medhq — APPROVED** [review]
nice work

**Review by IsraaAmani — APPROVED** [review]
well done

File: index.html [hunk 1]
```diff
@@ -56,10 +56,10 @@ <h2 >
...
     </h2>
     <div class="category">
-      <button id="abstract-btn" type="submit">Abstract</button>
-      <button id="nature-btn" type="submit">Nature</button>
-      <button id="cosmo-btn" type="submit">Cosmo</button>
-      <button id="minimal-btn" type="submit">Minimal</button>
+      <button id="abstract-btn" name="Abstract" type="submit">Abstract</button>
+      <button id="nature-btn" name="Nature" type="submit">Nature</button>
+      <button id="cosmo-btn" name="Cosmo" type="submit">Cosmo</button>
+      <button id="minimal-btn"  name="Minimal" type="submit">Minimal</button>
```
Comment (Ibrahim-Jarada) [inline index.html]: There is no need to add name attribute by the button content in order to get it in JS
You can access it directly by this code `event.target.textContent`

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const cosmoBtn = document.getElementById("cosmo-btn");
+const minimalBtn = document.getElementById("minimal-btn");
+abstractBtn.addEventListener("click", buttonName)
+natureBtn.addEventListener("click", buttonName)
+cosmoBtn.addEventListener("click", buttonName)
+minimalBtn.addEventListener("click",buttonName)
+
+
+function buttonName(event){
+    localStorage.setItem('name', event.target.name)
```
Comment (Ibrahim-Jarada) [inline js/index.js]: You can replace `event.target.name` by  `event.target.textContent`

**Review by Ibrahim-Jarada — CHANGES_REQUESTED** [review]

File: js/index.js (hunk 2 above)
Comment (kakashii7) [reply js/index.js]: thank you <3

**Review by Ibrahim-Jarada — APPROVED** [review]

**Pull request merged**